moduleroot/Gemfile

```diff
@@ -127,8 +139,14 @@ if explicitly_require_windows_gems
 end
 <% end -%>
 
+# Evaluate Gemfile.local if it exists
 if File.exists? "#{__FILE__}.local"
   eval(File.read("#{__FILE__}.local"), binding)
 end
 
+# Evaluate ~/.gemfile if it exists
```

Comment:
And usually my gemfile.local is the same across all gems, so why not just have one in my home directory?